### ISSUE TITLE
ARM64 / Apple Silicon Export Target for Max, VST etc. 

### ISSUE THREAD

**Issue by mmtrx-ai** [grame-cncm/faustide#73]
OSX platform target is not compatible with ARM64 / Apple Silicon, tested with vst(i), max.

**Comment by noahneumark** [comment]
Yeah I ran into the same issue compiling for Max. The result worked on my old Mac but not on Apple Silicon. Is this issue being addressed?
